src/components/bootcamps/BootcampEditForm.tsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { useRouter } from 'next/router'
+import { FormikHelpers } from 'formik'
+import { useMutation } from 'react-query'
+import { useTranslation } from 'next-i18next'
+import { AxiosError, AxiosResponse } from 'axios'
+import { AlertStore } from 'stores/AlertStore'
+import GenericForm from 'components/common/form/GenericForm'
+import SubmitButton from 'components/common/form/SubmitButton'
+import FormTextField from 'components/common/form/FormTextField'
+import { ApiErrors, isAxiosError, matchValidator } from 'common/api-errors'
+import { editBootcamp } from 'common/rest'
+import { Button, Container, Grid, Typography } from '@mui/material'
+import Layout from './layout/Layout'
+import { routes } from 'common/routes'
+import React from 'react'
+import { BootcampFormData, BootcampResponse, EditBootcampProp } from 'gql/bootcamps'
```

Comment:
Usually, the imports are being sorted into three groups by their source location and separated with an empty line. 

1. package imports
2. absolute imports
3. local imports

```suggestion
import React from 'react'
import { FormikHelpers } from 'formik'
import { useRouter } from 'next/router'
import { useMutation } from 'react-query'
import { useTranslation } from 'next-i18next'
import { AxiosError, AxiosResponse } from 'axios'
import { Button, Container, Grid, Typography } from '@mui/material'

import { routes } from 'common/routes'
import { editBootcamp } from 'common/rest'
import { AlertStore } from 'stores/AlertStore'
import GenericForm from 'components/common/form/GenericForm'
import SubmitButton from 'components/common/form/SubmitButton'
import FormTextField from 'components/common/form/FormTextField'
import { ApiErrors, isAxiosError, matchValidator } from 'common/api-errors'
import { BootcampFormData, BootcampResponse, EditBootcampProp } from 'gql/bootcamps'

import Layout from './layout/Layout'
```